pom.xml

```diff
@@ -41,6 +41,19 @@
         </snapshotRepository>
     </distributionManagement>
 
+    <repositories>
+        <repository>              
+            <id>alfresco-public</id>
+            <url>https://artifacts.alfresco.com/nexus/content/groups/public</url>
+            <layout>default</layout>
+        </repository>
+        <repository>
+            <id>alfresco-public-snapshots</id>
+            <url>https://artifacts.alfresco.com/nexus/content/groups/public-snapshots</url>
+            <layout>default</layout>
+        </repository>
+    </repositories>  
```

Comment:
Just replace this section with:
```
    <repositories>
        <repository>              
            <id>alfresco-internal</id>
            <url>https://artifacts.alfresco.com/nexus/content/groups/public</url>
            <layout>default</layout>
        </repository>
    </repositories>
```
That should be enough (the `alfresco-internal` label with the `../groups/public` url).

If that doesn't solve the issues, we'll have to review the artifacts that actually get pulled by this community project.